app/javascript/i18n-js/translations.js

```diff
@@ -0,0 +1,4 @@
+import { en } from "../../../config/locales/simulator/en.yml"
```

Comment:
This still has dependency on rails i18n mapping cc @satu0king

Reply:
exactly how ? so that i can understand concept behind it and opt for solutions.

If we dont want to import translations from yml to json . then we can directly prepare json files for translations and can dump into ```I18n.translations```

Reply:
Any updates here @satu0king @tachyons ?

Reply:
@pavanjoshi914, yeah we don't want any relationship with rails i18n mapping. To make it simple, all the files related to the simulator should be in the`simulator/src` folder. 

The `.yml` files can be used as long as it is in the `simulator` directory. Because this directory will be moved away in the future to become a standalone repo. Does that make sense now?

Reply:
Yaa. I used yml only with yml loader only following rails conventions.

If I prepare json file (no requirement of yml loader) and dump them into ``|I18n.translations```  present in config.js in a similar fashion using import statements .. 

Does it will work?

Reply:
If not.... In any case I am opting  for banana i18n library by wikimedia... That might help us in creating better solution...